### PR TITLE
Update links to RGAA 4.1.2 (and Opquast rules)

### LINKED ISSUE #443: Update links to RGAA criteries
The web addresses for the RGAA 3 criteries no longer exist. It is also not possible to refer to specific RGAA 4 criteria because the criteria headings on https://www.numerique.gouv.fr/publications/rgaa-accessibilite/methode-rgaa/criteres/#contenu do not have identifiers. Would it not be a good idea to update the addresses, using the page published by tanaguru https://rgaa.tanaguru.com/?

### PR DESCRIPTION
**Pull request by PhilippeVay**
Should close #443

#### tl;dr
* This PR updates RGAA links to the "official" up to date references (to themes, criterias mainly, tests and glossary entries) v4.1.2
* It also corrects 2-3 references to "BP160 Opquast".
* Documentation have been rebuilt, so it takes into account numerous modifications to WCAG links from PR #448 (I don't think they were versioned and it confused me a lot because I was discovering the directory structure and didn't pay attention to the directory of the files ^^) and other recent modifications.

#### RGAA
French RGAA has been updated to version 4.1.2 (fixing a regression from 4.1.1). The official website has a stable URL https://accessibilite.numerique.gouv.fr/, same for its technical part https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/ so it's safe to link to it.

In detail, `id` to a couple of Themes (_Thématiques_) have been manually replaced to `#5` or `#10`. Same for a couple of Glossary entries.  
`id` of **Tests** and **criterias** have been mass-replaced with those RegExp:

```
# Tests

From: https://rgaa.tanaguru.com/rgaa4-criteres.html#test-([1-9][0-9]?)-([0-9])-([0-9])
To  : https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#$1.$2.$3

# Criterias

From: https://rgaa.tanaguru.com/rgaa4-criteres.html#crit-([1-9][0-9]?)-([0-9]+)
To  : https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#$1.$2
```

#### Opquast
2-3 references to "BP160 Opquast" were unrelated to a11y.css. I guess it should've been n°5 and n°111.  
Also Opquast project chose to rename its BP ("Best Practice" or "_Bonnes Pratiques_" in french) to Rules ("_Règles_") so the references are Rule 5 and Rule 111.

### CODE REVIEW

File: docs/errors.html [hunk 1]
```diff
@@ -272,12 +271,12 @@ <h3>Description</h3>
 By the way, why would you use an empty link?</p>
 <h3>References</h3>
 <ul>
-<li><a href="https://references.modernisation.gouv.fr/rgaa/criteres.html#crit-6-5">https://references.modernisation.gouv.fr/rgaa/criteres.html#crit-6-5</a></li>
-<li><a href="https://www.w3.org/TR/WCAG20/#text-equiv-all">https://www.w3.org/TR/WCAG20/#text-equiv-all</a></li>
-<li><a href="https://www.w3.org/TR/WCAG20/#navigation-mechanisms-refs">https://www.w3.org/TR/WCAG20/#navigation-mechanisms-refs</a></li>
-<li><a href="https://www.w3.org/TR/WCAG20/#navigation-mechanisms-link">https://www.w3.org/TR/WCAG20/#navigation-mechanisms-link</a></li>
-<li><a href="https://www.w3.org/TR/WCAG-TECHS/H30.html">https://www.w3.org/TR/WCAG-TECHS/H30.html</a></li>
-<li><a href="https://www.w3.org/TR/WCAG-TECHS/G91.html">https://www.w3.org/TR/WCAG-TECHS/G91.html</a></li>
+<li><a href="https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#6.5">https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#6.5</a></li>
```
Comment (ffoodd) [inline docs/errors.html:274]: I'd need to test each links because this anchor doesn't existe, should be 6.2.

I think Linkinator cannot handle anchors :/

**Review by ffoodd — COMMENTED** [review]
Thanks a lot for this PR, it's really helpful! 

There're a few things I'd like to check before merging.

Give me a few days to check all of this 💜

File: docs/errors.html (hunk 1 above)
Comment (PhilippeVay) [reply docs/errors.html:274]: Oops, found 3 others that can't exist

**Comment by PhilippeVay** [comment]
Re inline comment about `#6.5` (there are only 2 criterias for Links theme 6.x in RGAA)

This RegExp found 3 other references in `sass/themes/_warning.scss`: **1.10, 4.18 and 13.17**
```
#(1\.1[0-9]|2\.[3-9]|3\.[4-9]|4\.1[4-9]|5\.([9]|1[0-9])|6\.[3-9]|7\.[6-9]|8\.1[1-9]|9\.[5-9]|10\.1[5-9]|11\.1[4-9]|12\.1[2-9]|13\.1[3-9])
```
Images: 1.9 max (down from 10 or 11 in RGAA 3, in part because AAA criterias were first-class citizens 😛)
Frames: 1.2
Color: 3.3
Media: 4.13
Tables: 5.8
Links: 6.2
Scripts: 7.5
Mandatory elements: 8.10
Struct.: 9.4
Presentation: 10.14
Forms: 11.13
Navigation: 12.11
_Other:_ 13.12

I'll update those later.  
Definitely need a double or triple check though, and it's a long task 🕙

**Comment by PhilippeVay** [comment]
Done. There were also a 12.10 and 4.20 that my regex didn't catch.

**Comment by zwiastunsw** [comment]
All links in PR #458 have been very carefully checked and are correct. They lead to the RGAA version published by Tanaguru. This is, in my opinion, a better solution, because Tanagaru will probably, also publish an English translation of the RGAA

**Comment by ffoodd** [comment]
@zwiastunsw Tanaguru only publishes a RGAA mirror, so if those tests don't exist in the RGAA they cannot exist in Tanaguru's mirror — see [6.5 that the `main` branch currently exposes](https://rgaa.tanaguru.com/rgaa4-criteres.html#crit-6-5)… Also I tend to prefer the official source, as I commented in #443.

Since criterias and tests are anchors, we'll always get a `200` status. Not quite sure about how to handle this (yet), but there were only a few weeks between your PRs and it's already quite hard to maintain. I need to automate this in some way.

**Comment by ffoodd** [comment]
Thanks for all the hard work from both of you, I'll merge this as-is and try to find  way to check for anchor existence (#467).
